coda/coda_oaipmh/views.py

```diff
@@ -29,5 +29,5 @@ def index(request):
         resumption_batch_size=500
     )
 
-    response = oaiServer.handleRequest(request.REQUEST)
+    response = oaiServer.handleRequest(request.POST)
```

Comment:
I suspect this should also be a `request.GET` not `request.POST`. To help with checking this, I think it would be good to [test this view](https://github.com/unt-libraries/coda/blob/acbd6fdb95c208b584fe6a468472f3308b50fda6/coda/coda_oaipmh/tests/test_views.py#L9) more than just test the content_type and response code.

I notice that http://localhost:8787/oai/?verb=Identify when using `request.POST` we get XML and 200, but the response is:
```
?xml version='1.0' encoding='UTF-8'?>
<OAI-PMH xmlns="http://www.openarchives.org/OAI/2.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/ http://www.openarchives.org/OAI/2.0/OAI-PMH.xsd">
  <responseDate>2019-08-29T22:38:03Z</responseDate>
  <request>http://example.com/oai/</request>
  <error code="badVerb">Required verb argument not found.</error>
</OAI-PMH>
```
with a `request.GET` the response is again XML and 200, but the content is more like:
```
<?xml version='1.0' encoding='UTF-8'?>
<OAI-PMH xmlns="http://www.openarchives.org/OAI/2.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/ http://www.openarchives.org/OAI/2.0/OAI-PMH.xsd">
  <responseDate>2019-08-29T22:41:55Z</responseDate>
  <request verb="Identify">http://example.com/oai/</request>
  <Identify>
    <repositoryName>example.com</repositoryName>
    <baseURL>http://example.com/oai/</baseURL>
    <protocolVersion>2.0</protocolVersion>
    <adminEmail>mark.phillips@unt.edu</adminEmail>
    <earliestDatestamp>2004-05-19T00:00:00Z</earliestDatestamp>
    <deletedRecord>transient</deletedRecord>
    <granularity>YYYY-MM-DDThh:mm:ssZ</granularity>
    <description>
      <toolkit xmlns="http://oai.dlib.vt.edu/OAI/metadata/toolkit" xsi:schemaLocation="http://oai.dlib.vt.edu/OAI/metadata/toolkit http://oai.dlib.vt.edu/OAI/metadata/toolkit.xsd">
        <title>pyoai</title>
        <version>2.4.4</version>
        <URL>http://infrae.com/products/oaipack</URL>
      </toolkit>
    </description>
  </Identify>
</OAI-PMH>
```